src/RecoveryServices/RecoveryServices.Backup.Providers/Providers/AzureFilesPsBackupProvider.cs

```diff
@@ -499,7 +499,7 @@ public void DeletePolicy()
             {
                 // Container is not discovered. Throw exception
                 string errorMessage = string.Format(
-                    Resources.DiscoveryFailure,
```

Comment:
Hi @sambitratha since you add a new variable `AFSDiscoveryFailure`, you may want to remove `DiscoveryFailure` if there is no reference any more.

Reply:
@wyunchi-ms  DiscoveryFailure variable is referred IaasVM provider.

Reply:
OK